feature-group-definitions/server-timing.dist.yml

```diff
@@ -0,0 +1,25 @@
+# Generated from: server-timing.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Server timing
+spec: https://w3c.github.io/server-timing/
+caniuse: server-timing
+status:
+  baseline: low
+  baseline_low_date: 2023-03-27
+  support:
```

Comment:
This is a straightforward case of everything being the same versions:
https://developer.mozilla.org/en-US/docs/Web/API/PerformanceServerTiming#browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing#browser_compatibility

Reply:
Like finding a four-leaf clover 😆